include/megaapi.h

```diff
@@ -18285,6 +18318,25 @@ class MegaApi
         */
         void sendBackupHeartbeat(MegaHandle backupId, int status, int progress, int ups, int downs, long long ts, MegaHandle lastNode);
 
+        /**
+         * @brief Allow fetch google ads
+         *
+         * @param adFlags A bitmap flag used to communicate with the API
+         * @param adUnits A list of the adslot ids to fetch
+         * @param publicHandle Provide the public handle that the user is visiting (optionally)
```

Comment:
Since it's optional, it could be useful to set a default value in the signature (aka. `publicHandle = INVALID_HANDLE`).
Same for the other interface.